src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurExpenseV3ServiceImpl.java

```diff
@@ -167,9 +165,22 @@ protected void updateStatusInConcur(String accessToken, String reportId, boolean
         String logMessageDetail = buildLogMessageDetailForExpenseWorkflowAction(workflowAction, reportId);
         ConcurWebRequest<Void> webRequest = buildWebRequestForExpenseWorkflowAction(
                 workflowAction, reportId, resultsDTO);
+        
+        /*
+         * @todo re enable this
+         */
+        if (shouldUpdateConcur(reportId)) {
+            LOG.info("updateStatusInConcur, updating the status in Concur for report " + reportId);
+            concurEventNotificationWebApiService.callConcurEndpoint(accessToken, webRequest, logMessageDetail);  
+        } else {
+            LOG.error("updateStatusInConcur. updates disabled, but would have updated the status in Concur for report " + reportId);
+        }
 
-        concurEventNotificationWebApiService.callConcurEndpoint(
-                accessToken, webRequest, logMessageDetail);
+    }
+    
+    private boolean shouldUpdateConcur(String reportId) {
+        //return StringUtils.equalsIgnoreCase(reportId, "D9B34CBBA618434F9C34") || StringUtils.equalsIgnoreCase(reportId, "E9EA2595C7EC4CBF88C3");
+        return true;
```

Comment:
this is the code I mentioned in the PR description about local debug code that needs to be removed before the PR can be merged.
it can be removed now